### ISSUE TITLE
Percentile calculations seem to be off

### ISSUE THREAD

**Issue by fivenp** [kubeshop/testkube#2994]
**Describe the bug**

Having these 6 execution times: 

```
5.34
5.75
5.93
6.16
6.56
11.56
```

Testkube tells me my P95 is `6.56`
![image](https://user-images.githubusercontent.com/132332/210810054-0caaf3be-de8f-459a-8077-9af7be022d22.png)

Whereas the actual P95 (even P85!) is actually `11.56`
Multiple percentile calculators throw out the same result
![image](https://user-images.githubusercontent.com/132332/210810150-b4f2f136-86ca-442d-b8e5-9fbdc4f11ffd.png)

The real issue here is the message/assumption Testkube is sending to users "Everything is fine" – where in reality there is really an issue with the executions because P95 actually is DOUBLE P50

**Comment by scott-david-walker** [comment]
I just looked a little bit into this and found that small data sets on the `quantile` package seems to be inaccurate (never actually queries the percentile, just returns the value at an index of the samples.

I've made an issue about this on the [repo](https://github.com/bmizerany/perks/issues/24) but based on the last open issues and pr being ignored I wonder if they're maintaining it.

As a workaround, we could force a flush of the stream therefore forcing the internal func `query` (the actual percentile logic) to be called by doing

```
    q.Merge(q.Samples())
```

which will give more accuracy but sacrificing performance of the library (according to the comments in `stream.go`) but also feels a little hacky.

Would appreciate advice before going forward.

Cheers
